draft-ietf-tls-esni.md

```diff
@@ -884,6 +881,24 @@ in order to force wasteful decryption. Servers that support this feature should,
 for example, implement some form of rate limiting mechanism to limit the damage
 caused by such attacks.
 
+## Outer ClientHello
+
+Any information that the client includes in the ClientHelloOuter is visible to
+passive observers. The client SHOULD NOT send values in the ClientHelloOuter
+which would reveal a sensitive ClientHelloInner property.
```

Comment:
```suggestion
which would reveal a sensitive ClientHelloInner property, such as the true
server name. It MAY send values associated with the public name in the 
ClientHelloOuter.
```